HollowKnightComponent.cs

```diff
@@ -270,6 +287,8 @@ public class HollowKnightComponent {
                 case SplitName.BroodingMawlek: shouldSplit = mem.PlayerData<bool>(Offset.killedMawlek); break;
                 case SplitName.CityOfTears: shouldSplit = mem.PlayerData<bool>(Offset.visitedRuins); break;
                 case SplitName.Collector: shouldSplit = mem.PlayerData<bool>(Offset.collectorDefeated); break;
+                case SplitName.CollectorAndGrubs:

```

Comment:
We already have `GrubCityCollectorAll` which does the `scenesGrubRescued` check. Would that suffice for this use case?